packages/components/psammead-grid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-grid",
-  "version": "3.0.7",
+  "version": "3.0.8",
```

Comment:
Is this meant to be a patch bump? At the very least I would have thought minor (if you want to class it as a backwards compatible bug-fix), but it also alters the output, which has the potential to be a breaking change.

Reply:
hmm, I saw it as a 'bug fix' so went for a patch; I see minor as 'adding a new feature but not a breaking change'. Happy do minor instead or even major, not precious about it.

Reply:
This is now a minor bump, I hope this will be more agreeable now @simonsinclair ? The I have added new configuration but the change is completly backwards compatible.